lib/src/bubble_painter.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:floating_bubbles/src/bubble_animation.dart';
+import 'package:floating_bubbles/src/bubbles.dart';
+import 'package:flutter/material.dart';
+import 'package:simple_animations/simple_animations.dart';
+
+class BubblePainter extends CustomPainter {
+  final BubbleShape shape;
+  final double sizeFactor;
+  final double strokeWidth;
+  final int colorAlpha;
+  final List<BubbleAnimation> bubbles;
+  final PaintingStyle paintingStyle;
+
+  BubblePainter({
+    required this.bubbles,
+    required this.colorAlpha,
+    required this.paintingStyle,
+    required this.shape,
+    required this.sizeFactor,
+    required this.strokeWidth,
+  });
+
+  @override
+  void paint(Canvas canvas, Size size) {
+    final paint = Paint()
+      ..strokeWidth = strokeWidth
+      ..style = paintingStyle;
+
+    bubbles.forEach((bubble) {
```

Comment:
replace forEach with a for loop: https://dart-lang.github.io/linter/lints/avoid_function_literals_in_foreach_calls.html